.github/workflows/agent.yml

```diff
@@ -0,0 +1,32 @@
+name: Build agent packages.
+
+on: [push]
+
+env:
+  DOCKER_BUILDKIT: 1
+
+
+jobs:
+  unit-tests:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ "ubuntu-20.04", "windows-2019", "macos-11" ]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install python.
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8.10
+
+      - name: Install python dependencies
+        run: |
+          python3 -m pip install -r dev-requirements.txt
```

Comment:
We probably also want to cache the dependencies to speed up the build, right?

Reply:
Right, There is already logic for that. I just didn't put it right now to reduce the number of new files, but it has already been added in the parallel PR with new docker images https://github.com/scalyr/scalyr-agent-2/pull/794.
In both PRs they have not been used fully yet, but in the next PR, they are used exactly for the reason that you've noticed here.